static/app/views/settings/account/accountIdentities.spec.jsx

```diff
@@ -74,9 +73,10 @@ describe('AccountIdentities', function () {
 
     expect(mock).not.toHaveBeenCalled();
 
-    wrapper.find('Button').first().simulate('click');
-    const modal = await mountGlobalModal();
-    modal.find('Button[priority="danger"]').simulate('click');
+    userEvent.click(screen.getByRole('button', {name: 'Disconnect'}));
+
+    render(<GlobalModal />);
```

Comment:
Two renders? Seems weird but I guess it works? I've done `<Fragment><AccountIdentities /><GlobalModal /></Fragment>` when I needed this before

Is there a reason why we don't render `<GlobalModal />` with the rest of the providers? Seems like that would be the most correct and less work when writing tests.

Reply:
I'm indifferent, both work right. We probably need to pick a pattern for this